### PR TITLE
Added file-a.md

### PR DESCRIPTION
**Pull request by Selim-ElChoueiter**

### CODE REVIEW

File: file-a.md [hunk 1]
```diff
@@ -0,0 +1 @@
+1 + 1 = 3
```
Comment (ras444) [inline file-a.md:1]: 1+1=2 not 3

**Comment by ras444** [comment]
1+1=2 not 3

File: file-a.md (hunk 1 above)
Comment (Selim-ElChoueiter) [reply file-a.md]: It has been fixed

Comment (ras444) [reply file-a.md]: tamem

**Pull request merged**